org.lflang/src/org/lflang/cli/CliBase.java

```diff
@@ -252,4 +316,54 @@ public Resource getResource(Path path) {
         }
     }
 
+    private String[] jsonStringToArgs(String jsonString) {
+        ArrayList<String> argsList = new ArrayList<>();
+        JsonObject jsonObject = new JsonObject();
+
+        // Parse JSON string and get top-level JSON object.
+        try {
+            jsonObject = JsonParser.parseString(jsonString).getAsJsonObject();
+        } catch (JsonParseException e) {
+            reporter.printFatalErrorAndExit(
+                    "Invalid JSON string:\n" + jsonString);
```

Comment:
```suggestion
                    String.format("Invalid JSON string:%n %s", jsonString));
```
Note that this renders line breaks properly across platforms.